src/core/issues-notes.js

```diff
@@ -146,6 +159,36 @@ function handleIssues($ins, ghIssues, conf) {
   }
 }
 
+//derives the text-color-class based on the illumination score of the labelColor
+function deriveTextColorClass(hexColor) {
+  try {
+    const rgb = parseInt(hexColor, 16);
+    if (isNaN(rgb)) {
+      throw NaN;
+    }
+    const red = (rgb >> 16) & 0xff;
+    const green = (rgb >> 8) & 0xff;
+    const blue = (rgb >> 0) & 0xff;
+    const illumination = 0.2126 * red + 0.7152 * green + 0.0722 * blue;
+    return illumination > 140 ? "light" : "dark";
+  } catch (err) {
+    console.error(err);
```

Comment:
Maybe we can improve the text content of `console.error()`

Reply:
Remember that console.error() is not for end-users, it's just for ReSpec maintainers so we can debug any issues. Generally, we only need `err` and the stack trace to figure out what happened.